### ISSUE TITLE
Dataset issues

### ISSUE THREAD

**Issue by TongjiLxy** [AudioVisualEmotionChallenge/AVEC2019#9]
Hello, I encountered a problem while reproducing the cross-cultural sub project (CES) of this project. I was unable to find the accurate training set, testing set, and validation set. The description of the dataset in Table 3 of the paper is video chat, so the activity chose video chat;
Data training and validation sets require labels such as Valence, Arousal, and Liking. Three datasets are available for all three labels: 30 in Germany and 35 in Hungary, but in the paper, there are only 34 training sets, plus 14 validation sets.
Can you tell me how to select data and partition them? It would be even better if a dataset could be provided. My email is 2210458@tongji.edu.cn . Looking forward to your reply.
